pyshell

```diff
@@ -1,2 +1,2 @@
 #!/bin/sh
-echo "" | dmenu -i -p "Python" | xargs -0 -I % python -c "import random;print(%)" | xargs -I % notify-send "Result" "%"
+printf '' | dmenu -i -p 'Python' | xargs -I % python -c "import random;print(%)" | xargs -I % notify-send 'Result' "%"
```

Comment:
In the original code, `echo` was appending a newline character, whereas `printf` isn't, so as long as that doesn't break something later on in the pipeline, I'm all for this; failing that, add a `\n` to `printf`, and -- I assume -- making the needed changes to `xargs`, regarding the `-0`.

I feel like this whole line is better written another way, if `xargs` has to be called twice in the same pipeline, but with `dmenu`, this may be the only sensible way, at least for now.